code/game/objects/machinery/camera/camera.dm

```diff
@@ -296,7 +295,7 @@
 	if(on)
 		set_light(AI_CAMERA_LUMINOSITY, AI_CAMERA_LUMINOSITY)
```

Comment:
wat, it's `set_light(l_range, l_power,...)` this doesn't make sense

Reply:
Both are 6

Reply:
and? still dumb AF, the defines are being abused